pagination/keysetpagination/paginator.go

```diff
@@ -62,7 +70,8 @@ func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 	return func(q *pop.Query) *pop.Query {
 		return q.
 			Limit(p.Size()+1).
-			Where(fmt.Sprintf("%s > ?", id), p.Token())
+			Where(fmt.Sprintf(`"%s" > ?`, id), p.Token()).
+			Order(fmt.Sprintf(`"%s" %s`, id, p.SortDirection()))
```

Comment:
```suggestion
			Where(fmt.Sprintf(`%q > ?`, id), p.Token()).
			Order(fmt.Sprintf(`%q ASC`, id))
```

This automatically adds double-quotes :wink: